mlflow/projects/__init__.py

```diff
@@ -695,9 +696,17 @@ def _get_docker_command(image, active_run):
                                  experiment_id=active_run.info.experiment_id)
     tracking_uri = tracking.get_tracking_uri()
     local_path, container_tracking_uri = _get_local_uri_or_none(tracking_uri)
+    artifact_uri_local_path = get_local_path_or_none(active_run.info.artifact_uri)
     if local_path is not None:
         cmd += ["-v", "%s:%s" % (local_path, _MLFLOW_DOCKER_TRACKING_DIR_PATH)]
         env_vars[tracking._TRACKING_URI_ENV_VAR] = container_tracking_uri
+    if artifact_uri_local_path is not None:
+        container_path = artifact_uri_local_path
+        if not os.path.isabs(container_path):
+            container_path = os.path.join("/mlflow/projects/code/", artifact_uri_local_path)
```

Comment:
Would be nice to pull "/mlflow/projects/code/" into a constant so we don't have this string duplicated across `mlflow/projects/__init__.py`, if you want to make a follow-up PR to do that :)